sourcegraph/code-intel-extensions
# Bug fix: Make searchLSIFSupportRobust dismissible

## Context
As outlined in [the issue](https://github.com/sourcegraph/sourcegraph/issues/38482), we want all hints about the nature of search results to be dismissible. One of the "search results" wasn't.

This PR fixes it.

## How to test
1. Pull the branch
2. Run `yarn run generate --languages=go && yarn --cwd generated-go run serve` as we will be testing with a go-file
3. Pull `sourcegraph` and run `sg start web-standalone`
4. Side-load the extension as outlined here: https://github.com/sourcegraph/code-intel-extensions#development
5. Open https://sourcegraph.test:3443/github.com/tsenart/vegeta/-/blob/lib/plot/assets_vfsdata.go
6. Hover over `Assets` and check that it's dismissible indeed

<img width="927" alt="Screenshot 2022-07-12 at 15 11 07" src="https://user-images.githubusercontent.com/2196347/178486762-5624cf40-b451-4824-8bf5-f2300b721bac.png">
